lib/tabs.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:off_top_mobile/login.dart';
+import 'package:off_top_mobile/recordingSession.dart';
+import 'package:off_top_mobile/settings_page.dart';
+
+class Tabs extends StatelessWidget {
```

Comment:
move this to components/footer/ and consider renaming it to the something more readible like footerTabs or bottomNavigationTabs